config/HH_bbWW/Run3_2022/samples.yaml

```diff
@@ -1,6 +1,621 @@
+  mass: 260
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-270:
+  sampleType: GluGluToRadion
+  mass: 270
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-280:
+  sampleType: GluGluToRadion
+  mass: 280
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph  
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-300:
+  sampleType: GluGluToRadion
+  mass: 300
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-350:
+  sampleType: GluGluToRadion
+  mass: 350
+  spin: 0
+  crossSection: 1pb    
+  generator: madgraph
 GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-450:
   sampleType: GluGluToRadion
+  mass: 450
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph 
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-550:
+  sampleType: GluGluToRadion
+  mass: 550
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-600:
+  sampleType: GluGluToRadion
+  mass: 600
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-650:
+  sampleType: GluGluToRadion
+  mass: 650
+  spin: 0
+  crossSection: 1pb    
+  generator: madgraph
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-700:
+  sampleType: GluGluToRadion
+  mass: 700
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-800:
+  sampleType: GluGluToRadion
+  mass: 800
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-1000:
+  sampleType: GluGluToRadion
+  mass: 1000
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-1200:
+  sampleType: GluGluToRadion
+  mass: 1200
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-1400:
+  sampleType: GluGluToRadion
+  mass: 1400
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-1600:
+  sampleType: GluGluToRadion
+  mass: 1600
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-1800:
+  sampleType: GluGluToRadion
+  mass: 1800
+  spin: 0
+  crossSection: 1pb
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-2000:
+  sampleType: GluGluToRadion
+  mass: 2000
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-2500:
+  sampleType: GluGluToRadion
+  mass: 2500
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph 
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-3000:
+  sampleType: GluGluToRadion
+  mass: 3000
+  spin: 0
+  crossSection: 1pb 
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-4000:
+  sampleType: GluGluToRadion
+  mass: 4000
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph 
+GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-5000:
+  sampleType: GluGluToRadion
+  mass: 5000
+  spin: 0
+  crossSection: 1pb
+  generator: madgraph
+############## Signals(SL,S=2)##############
+GluGlutoBulkGravitontoHHto2B2Vto2B2JLNu_M-250:
+  sampleType: GluGluToGraviton
```

Comment:
GluGluToGraviton -> GluGluToBulkGraviton

Reply:
Good catch, it was indeed missing because GluGlutoHHto2B2V_kl-1p00_kt-1p00_c2-0p00 was written twice mistakenly